### ISSUE TITLE
Rocket trajectory example broken since v4.0.0

### ISSUE THREAD

**Issue by josdejong** [josdejong/mathjs#1136]
I noticed that the rocket_trajectory_optimization.html example is broken since v4.0.0. I haven't looked into it yet, it probably has to do with changes in implicit mulitplication.

https://github.com/josdejong/mathjs/blob/master/examples/browser/rocket_trajectory_optimization.html

To reproduce: open the example, change 
```
<script src="../../dist/math.js"></script>
```
to:
```
<script src="https://unpkg.com/mathjs@3.20.2/dist/math.js"></script> works
<script src="https://unpkg.com/mathjs@4.0.0/dist/math.js"></script> doesn't work
```

@ericman314 maybe it's easy for you to figure out the cause?

**Comment by josdejong** [comment]
I've found the cause and fixed it. Two occurrences of implicit multiplication of which the behaviour was changed in v4.0.0.

**Comment by josdejong** [comment]
This is fixed now in `v5.0.2`
